config/routes.rb

```diff
@@ -607,6 +607,10 @@
     end
   end
 
+  get "build_info", to: "build_info#index"
+  get "app_build_info", to: "build_info#app_build_info"
```

Comment:
See my comment above in the controller. Unless there's a need for `/app_build_info` to have a different path than `/admin/app_build_info`, they can be combined into the admin version to simplify things a little